packages/doctype-caip10-link/package.json

```diff
@@ -26,10 +26,11 @@
     "clean": "rm -rf ./lib"
   },
   "dependencies": {
-    "3id-blockchain-utils": "^1.0.0",
+    "3id-blockchain-utils": "^1.1.0",
     "@ceramicnetwork/common": "^0.12.1",
     "@types/lodash.clonedeep": "^4.5.6",
-    "cids": "^1.0.2"
+    "cids": "^1.0.2",
+    "@zondax/filecoin-signing-tools": "^0.12.0"
```

Comment:
Can we move this to the CLI? This will increase the size of the http-client package which we're trying to avoid here.

Reply:
Sure.